chef/cookbooks/keystone/recipes/ha.rb

```diff
@@ -65,5 +65,12 @@
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 
+  pacemaker_order "o-#{service_name}" do
+    ordering "g-haproxy g-postgresql g-rabbitmq cl-#{service_name}"
```

Comment:
Something like this should be enough:

```ruby
pacemaker_order "o-#{service_name}" do
  score "Mandatory"
  ordering "postgresql cl-#{service_name}"
  action [ :create ]
  only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
  only_if "crm configure show postgresql"
end
```

But it's very ugly, and we wanted to write a helper for this.